packages/linux-drivers/gpu-sunxi/package.mk

```diff
@@ -32,23 +32,12 @@ PKG_LONGDESC="gpu-sunxi: Linux drivers for Mali GPUs found in Allwinner SoCs"
 PKG_TOOLCHAIN="manual"
 PKG_IS_KERNEL_PKG="yes"
 
-DRIVER_DIR=$PKG_BUILD/src/devicedrv/mali/
-
-pre_patch() {
-  # move source dir to allow patching
-  mv $PKG_BUILD/driver/src $PKG_BUILD/
-  rm -rf $PKG_BUILD/driver
-}
+DRIVER_DIR=$PKG_BUILD/driver/src/devicedrv/mali/
```

Comment:
Could this be changed to `PKG_DRIVER_DIR`?

Reply:
sure